1주차/1059.py

```diff
@@ -0,0 +1,24 @@
+L = int(input())
+s = list(map(int, input().split()))
+s = sorted(s)
```

Comment:
이것도 아래처럼 간단하게 만들 수 있으니 참고하세요~
```suggestion
s = sorted([*map(int, input().split())])
```